config/tsconfig.base.json

```diff
@@ -24,6 +24,7 @@
         "removeComments": false,
         "sourceMap": false,
         "stripInternal": true,
-        "target": "es5"
+        "target": "es5",
+        "typeRoots": ["../node_modules/@types"]
```

Comment:
why is this necessary now? we've never needed it in the past.

Reply:
This was just something I had to add on my end because of a bug with `at-loader` on Windows. In particular, it was causing Typescript to complain about being unable to find `require()`. There's some discussion of it here: https://github.com/gdi2290/angular-starter/issues/1371
It shouldn't affect anything else, but happy to split this off into a separate PR to reduce noise in the diff.

Reply:
hmmm `core` [declares the `require()` type](https://github.com/palantir/blueprint/blob/develop/packages/core/src/components/index.ts#L7). we should probably just do the same in icons rather than changing the tsconfig.

Reply:
Ok, sounds good to me. However, this also fixes a slew of errors that occur as a result of that `at-loader` + Windows interaction. For example, it doesn't pick up the typings for Mocha, so compiling unit tests gives me a wall of `can't find "it/describe/etc"` errors. I'll definitely submit this as a separate PR then.